### ISSUE TITLE
Supportr Feature phone / Dumbphone

### ISSUE THREAD

**Issue by ketsuban152** [deckerst/aves#2224]
### Guidelines

- [x] I have read the [guidelines to open issues](https://github.com/deckerst/aves/issues/234)

### Use case

I am using an SH-02L (a feature phone), and I installed the Aves app on it. 

However, I am having some issues because the app does not seem to fully support D-pad navigation.

### Proposal

1. Even if I press KEYCODE_DPAD_LEFT or KEYCODE_DPAD_RIGHT, the screen will not switch to the next picture.
This is probably because pressing KEYCODE_DPAD_UP from the five buttons on the bottom bar doesn't move the focus to the picture.

2. I'd like to see a feature implemented that's similar to MX Player's auto-rotate function.

**Comment by deckerst** [comment]
I'm interested to support this. Besides this navigation issue, do you know whether there are other limitations on that kind of device?

**Comment by ketsuban152** [comment]
Thank you for your reply.

Here are the limitations of my SH-02L:
- While it is easier to use in Android TV Mode, I still cannot zoom in. Also, in the split-screen view, I cannot access the settings screen on the far right. Furthermore, there is no screen rotation button.
- There is no volume button. I sometimes see devices that support hotkeys via volume buttons, but this does not work on the SH-02L.
